packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/implementations/multi_method_channel.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+typedef _MultiMethodChannelHandler(MethodCall call);
+
+class MultiMethodChannel extends MethodChannel {
```

Comment:
Wondering if we should move this into a package (in the Flutter SDK? in the plugin_platform_interface package?) since it will be useful for firebase_database and firebase_storage. /cc @amirh